docs/admin/resources/lifecycle.md

```diff
@@ -3,4 +3,44 @@ sidebar_position: 2
 id: lifecycle
 title: "Release Lifecycle"
 draft: true
----
\ No newline at end of file
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Release Types
+
+OpenCloud offers three release types to suit different needs: **Rolling**, **Production**, and **LTS**. For most users, Rolling is ideal, providing the **latest features** every few weeks. Production focuses on **stability**. LTS (Long-Term Support Release) is designed specifically for businesses that need longevity and extended **backports**. 
+
+<img src={require("./img/lifecycle/Release Cycle OpenCloud.png").default} alt="Release types" width="1920"/>
+
+
+### Advantage of the LTS Releases
+
+With LTS, businesses can continue using an older production release without needing to upgrade to the latest version, while still **receiving critical security patches** and **critical stability fixes**. This makes LTS the perfect choice for organizations seeking a **stable long-term** solution. LTS is available exclusively to customers with a service and support entitlement through a business license.
+
+| Rolling | Production | LTS |
+|:--------|:-----------|:----|
+| <ul><li>For enthusiasts</li><li>Contains latest features</li><li>Every 3 weeks</li><li>Automated quality assurance</li><li>Community supported</li></ul> | <ul><li>For production</li><li>Focus on stability</li><li>About every 6 months</li><li>Full quality assurance</li><li>Professional support</li></ul> | <ul><li>For production</li><li>Focus on longevity</li><li>Backports for 2 years</li><li>Full quality assurance</li><li>Professional support</li></ul> |
```

Comment:
Please try to use markdown whenever possible

Reply:
yep, I tried using line breaks and dashes for the bullets, but that does not render as proper list item. so formatting in table cells seems to be limited. are you aware how to do this in a proper way?

```
| Rolling | Production | LTS |
|:--------|:-----------|:----|
| - bla asdbla asdbla asdbla asdbla asdbla asdbla asdbla asd <br /> - bla asdbla asdbla asdbla asdbla asasdbla asdbla asdbla asdbla asasdbla asdbla asdbla asdbla asasdbla asdbla asdbla asdbla asasdbla asdbla asdbla asdbla asdbla asdbla asdbla asd <br /> - bla asdbla asdbla asdbla asdbla asdbla asdbla asdbla asd <br /> |
```

![screenie000363@2x](https://github.com/user-attachments/assets/7194be8c-4c49-4eed-bfd4-b363b81113b4)